web/components/analytics/scope-and-demand/scope-and-demand.tsx

```diff
@@ -45,7 +45,7 @@ export const ScopeAndDemand: React.FC<Props> = (props) => {
 
   // scope data
   const pendingIssues = defaultAnalytics?.pending_issue_user ?? [];
-  const pendingUnAssignedIssues = pendingIssues?.filter((issue) => issue.assignees__id === null);
+  const pendingUnAssignedIssuesUser = pendingIssues?.find((issue) => issue.assignees__id === null);
```

Comment:
Find gives the first element. I don't think that is the behaviour you want here?

Reply:
As discussed offline, we need the first and only element.